QUICKSTART.md

```diff
@@ -0,0 +1,32 @@
+# Quick Start Guide
+
+## Pre-requisites
+
+* [Install Virtual Box 5.1.14 or later]( https://www.virtualbox.org/wiki/Downloads )
```

Comment:
These are just the latest versions. I think any version that works with the latest centos image should be fine.